sdk/apimanagement/azure-mgmt-apimanagement/azure/mgmt/apimanagement/_configuration.py

```diff
@@ -12,14 +12,13 @@
 from azure.core.pipeline import policies
 from azure.mgmt.core.policies import ARMHttpLoggingPolicy
 
-from ._version import VERSION
-
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
     from typing import Any
 
     from azure.core.credentials import TokenCredential
 
+VERSION = "unknown"
```

Comment:
Need add package_version back in readme.python.md so that `VERSION` is not `unknown`.